### DISCUSSION TITLE
Melodos

### DISCUSSION THREAD

**Discussion by Mykbibby** [neanes/neanes#1039] [Ideas]
Can't remember if this has been discussed yet, but would there be a way to create an import mechanism for Melodos scores?

It would be a great help, prior to support for OCR, because it would enable user to import existing Greek scores as a starting point for English compositions. There's a ton of Melodos content out there, and it would be great to be able to tap into that.

If there are legal issues preventing this, I get it, in which case any solution to importing pre-existing Greek scores would be highly appreciated.

**Comment by danielgarthur** [comment]
I looked at the `.mel` file format and it appears to be a binary serialization of .NET Framework classes. Without access to those class definitions, it would not be feasible to determine the file structure.

In addition, the Melodos software license contains a clause that forbids reverse engineering the application. So peeking inside their source code (which is probably obfuscated, anyway) is a no-go. Without collaboration with the Melodos team, this won't be possible.
